CHANGELOG.md

```diff
@@ -13,6 +37,10 @@ BUG FIXES
 
 - [consensus] Fix issue #1575 where a late proposer can get stuck
 
+BREAKING:
```

Comment:
should we remove this?

Reply:
oh guh. changelog merging never works properly